apps/tailwind-components/pages/DataFetch.other.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <h1>Data and Meta data</h1>
+  <div class="h-12 mt-4 mb-16">
+    <h3 class="text-heading-lg">Params</h3>
+    <div class="m-2">
+      <label for="schema-id-input">schema id: </label>
+      <select id="table-id-select" v-model="schemaId">
+        <option v-for="option in schemaOptions" :value="option">
+          {{ option }}
+        </option>
+      </select>
+    </div>
+    <div class="m-2">
+      <label for="table-id-select">table id: </label>
```

Comment:
Is this component used more than once? If so, we need to autogenerate the identifiers